src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/ConfigurationBinderBenchmarks.cs

```diff
@@ -0,0 +1,91 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Extensions.Configuration
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class ConfigurationBinderBenchmarks
+    {
+        public static IEnumerable<int> GetDuplicateCount()
+            => Enumerable.Range(1, 3).Select(i => 2 * i);
+
+        public static IEnumerable<int> GetMyObjectCount()
+            => Enumerable.Range(1, 3).Select(i => 3 * i);
```

Comment:
Isn't it easier to read and understand if you just put the 3 numbers in the code?

```suggestion
        public static IEnumerable<int> GetDuplicateCount() => new int[] { 2, 4, 6};

        public static IEnumerable<int> GetMyObjectCount() => new int[] { 3, 6, 9};
```